packages/illustrated-message/src/spectrum-illustratedmessage.css

```diff
@@ -12,61 +12,169 @@ governing permissions and limitations under the License.
 
 /* THIS FILE IS MACHINE GENERATED. DO NOT EDIT */
 :host {
-    --spectrum-illustratedmessage-description-max-width: 500px;
-    --spectrum-illustratedmessage-heading-max-width: 500px;
-    --spectrum-illustratedmessage-illustration-margin-bottom: 24px;
-    --spectrum-illustratedmessage-heading-margin: 0;
-    --spectrum-illustratedmessage-description-margin: 4px 0 0 0;
+    --spectrum-illustrated-message-description-max-inline-size: var(
+        --spectrum-illustrated-message-maximum-width
+    );
+    --spectrum-illustrated-message-heading-max-inline-size: var(
+        --spectrum-illustrated-message-maximum-width
+    );
+    --spectrum-illustrated-message-title-to-heading: var(
+        --spectrum-spacing-400
+    );
+    --spectrum-illustrated-message-heading-to-description: var(
+        --spectrum-spacing-75
+    );
+    --spectrum-illustrated-message-illustration-color: var(
+        --spectrum-neutral-visual-color
+    );
+    --spectrum-illustrated-message-illustration-accent-color: var(
+        --spectrum-accent-visual-color
+    );
+    --spectrum-illustrated-message-title-font-family: var(
+        --spectrum-sans-font-family-stack
+    );
+    --spectrum-illustrated-message-title-font-weight: var(
+        --spectrum-heading-sans-serif-font-weight
+    );
+    --spectrum-illustrated-message-title-font-style: var(
+        --spectrum-heading-sans-serif-font-style
+    );
+    --spectrum-illustrated-message-title-font-size: var(
+        --spectrum-illustrated-message-title-size
```

Comment:
I think we're good on this one. The design comment I received was: 
> This looks right to me – there’s a couple components where the mobile (“large”) scale is smaller than desktop (“medium”) scale. it’s mostly because the mobile screens are usually narrower, so if we make the text too big, it won’t fit in the screen.